wordle-web/app.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <NuxtLayout>
+    <v-app>
+      <v-btn @click="router.push('/')">Home</v-btn>
+      <v-btn @click="router.push('/test')">Test</v-btn>
```

Comment:
Nice job including the routers, but what is the `test` page used for? 
I would suggest making your `test` page a `game` page so that you can navigate between Home and Game as the assignment requires.

Reply:
Thank you for all the feedback!